### PR TITLE
Common/ROS: first draft of ap_navigation config

### PR DESCRIPTION
**Pull request by rmackay9**
This adds setup files for getting object avoidance working with ROS as described on [this wiki page](http://ardupilot.org/dev/docs/ros-object-avoidance.html).

### CODE REVIEW

File: Common/ROS/ap_navigation/package.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <!--   <build_export_depend>message_generation</build_export_depend> -->
+  <!-- Use buildtool_depend for build tool packages: -->
+  <!--   <buildtool_depend>catkin</buildtool_depend> -->
+  <!-- Use exec_depend for packages you need at runtime: -->
+  <!--   <exec_depend>message_runtime</exec_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <!-- Use doc_depend for packages you need only for building documentation: -->
+  <!--   <doc_depend>doxygen</doc_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
```
Comment (khancyr) [inline Common/ROS/ap_navigation/package.xml:27]: surely need at least : 
<exec_depend>move_base</exec_depend>
<exec_depend>mavros</exec_depend>
you should put here every ros package needed to make it works !
I think that tf2, costmap_2d are used too 
that allow to auto fetch the right dependecies with rosdep !

**Review by khancyr — COMMENTED** [review]
why put the files + the zip ?

**Comment by rmackay9** [comment]
@khancyr, thanks for the review.  I added the .zip to make it easier to install.  The [installation wiki page is here](http://ardupilot.org/dev/docs/ros-object-avoidance.html).

File: Common/ROS/ap_navigation/package.xml (hunk 1 above)
Comment (rmackay9) [reply Common/ROS/ap_navigation/package.xml:27]: added, thanks!

**Comment by rmackay9** [comment]
I should probably let @peterbarker merge this but I'd like to get it in to make the wiki instructions work (they rely on this getting into master)

**Pull request merged**